mlflow/tracking/client.py

```diff
@@ -520,6 +520,12 @@ def create_model_version(self, name, source, run_id, tags=None, run_link=None):
                 # Databricks Secret Manager with scope=<scope> and key=<prefix>-workspaceid.
                 workspace_host, workspace_id = \
                     get_workspace_info_from_databricks_secrets(tracking_uri)
+                if not workspace_id:
+                    print("No workspace ID specified; if your Databricks workspaces share the same"
+                          "host URL, you may want to specify the workspace ID (along with the host"
+                          "information in the secret manager) for run lineage tracking. For more"
+                          "details on how to specify this information in the secret manager,"
+                          "please refer to the model registry documentation.")
```

Comment:
i think you need a space between the consecutive lines? or can just do a large print block ("""...""") that will have some newlines in it.

Reply:
Added spaces! thanks for catching that